cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -206,6 +207,28 @@ def test_tag_param_returns_resources_tagged_that_way(self):
         )
 
 
+@unittest.skipIf(wagtail.VERSION < (2, 0), "No need to test in Wagtail 1+")
+class TestWhitelistOverride(SimpleTestCase):
+    # Borrowed from https://github.com/wagtail/wagtail/blob/master/wagtail
+    # /core/tests/test_whitelist.py
+
+    def test_whitelist_hooks(self):
+        """Test that Whitelister does not strip scan element and attributes.
+
+        The allowed element <scan> and attributes are added in v1.wagtail_hooks.
```

Comment:
Two tiny typos here.

```suggestion
        """Test that Whitelister does not strip span element and attributes.

        The allowed element <span> and attributes are added in v1.wagtail_hooks.
```

Reply:
Good catch. I will replace `scan` with `span` in comments.